torchvision/models/detection/transform.py

```diff
@@ -30,17 +30,19 @@ def __init__(self, min_size, max_size, image_mean, image_std):
         self.image_std = image_std
 
     def forward(self, images, targets=None):
+        transformed_images = []
```

Comment:
A simpler way of doing this (and a one-liner) would be
```python
images = [img for im in images]
```

Reply:
Done with list slicing.